bnd-maven-plugin-parent/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    
+    <groupId>biz.aQute.bnd</groupId>
+    <artifactId>bnd-maven-plugin-parent</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>bnd-maven-plugin-parent</name>
+    <description>This maven plugin can build and test bndtools.org projects created using
+         the bndtools Eclipse-Based developer tooling from the command line using Maven</description>
+    <url>http://github.com/bndtools/bnd/blob/master/bnd-maven-plugin-parent/README.txt</url>
+    
+    <licenses>
+        <license>
+            <name>Apache Software License 2.0</name>
+            <url>http://www.opensource.org/licenses/apache2.0.php</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <url>https://github.com/bndtools/bnd</url>
+        <connection>scm:git:https://github.com/bndtools/bnd.git</connection>
+        <developerConnection>scm:git:https://github.com/bndtools/bnd.git</developerConnection>
+      <tag>HEAD</tag>
+  </scm>
+
+    <developers>
+        <developer>
+            <id>njbartlett</id>
+            <name>Neil Bartlett</name>
+            <organization>Paremus Ltd</organization>
+        </developer>    
+    </developers>
+
+    <properties>
+        <bnd.version>2.5.0-SNAPSHOT</bnd.version>
```

Comment:
Ah. Here is a variable with the bnd maven plugin version. We need to use the var in all the other pom files.

It should also be 3.0.0-SNAPSHOT.